cypress/support/HomeWork.PO.ts

```diff
@@ -0,0 +1,16 @@
+///<reference types="Cypress" />
+
+import {BaseComponent} from "./Base.component";
+
+export class HomeWorkPO extends BaseComponent{
```

Comment:
HomeWork is not a page. PO pattern will work if the name of your object will be as your page name